Makefile

```diff
@@ -43,6 +43,8 @@ ISORT_PARAMS = --ignore-whitespace --settings-path ./ --recursive raiden/ -sg */
 
 lint:
 	flake8 raiden/ tools/
+	ls
```

Comment:
What's this ls doing here? :)

Reply:
it's from the test commit, I don't know why the build is failling